pkg/libovsdb/schema.go

```diff
@@ -1,29 +1,37 @@
 package libovsdb
 
 import (
+	"encoding/json"
 	"fmt"
 	"io"
+	"strings"
 )
 
+type Schema map[string]*DatabaseSchema
+
 // DatabaseSchema is a database schema according to RFC7047
 type DatabaseSchema struct {
 	Name    string                 `json:"name"`
 	Version string                 `json:"version"`
 	Tables  map[string]TableSchema `json:"tables"`
```

Comment:
Checksum is missing `json:"cksum,omniempty"`

Reply:
ok